include/nbl/builtin/hlsl/blur/common.hlsl

```diff
@@ -0,0 +1,87 @@
+#pragma once
+
+#ifndef _NBL_BOX_BLUR_COMMONS_
+#define _NBL_BOX_BLUR_COMMONS_
+
+#include <nbl/builtin/hlsl/cpp_compat.hlsl>
+#include <nbl/builtin/hlsl/cpp_compat/intrinsics.h>
+#include <nbl/builtin/hlsl/glsl_compat/core.hlsl>
+
+//_NBL_STATIC_INLINE_CONSTEXPR uint32_t WORKGROUP_SIZE = 256u;
+//_NBL_STATIC_INLINE_CONSTEXPR uint32_t PASSES_PER_AXIS = 3u;
+//_NBL_STATIC_INLINE_CONSTEXPR uint32_t AXIS_DIM = 3u; // HUH ?
+
+enum WrapMode : uint32_t
+{
+	WRAP_MODE_CLAMP_TO_EDGE,
+	WRAP_MODE_CLAMP_TO_BORDER,
+	WRAP_MODE_REPEAT,
+	WRAP_MODE_MIRROR,
+};
+
+enum BorderColor : uint32_t
+{
+	BORDER_COLOR_FLOAT_TRANSPARENT_BLACK,
+	BORDER_COLOR_INT_TRANSPARENT_BLACK,
+	BORDER_COLOR_FLOAT_OPAQUE_BLACK,
+	BORDER_COLOR_INT_OPAQUE_BLACK,
+	BORDER_COLOR_FLOAT_OPAQUE_WHITE,
+	BORDER_COLOR_INT_OPAQUE_WHITE
+};
```

Comment:
maybe lets move the same enums defined in `nbl::asset` to a common HLSL header and alias the old enums in ISampler to those?